src/components/charts/MonthlyTotalHistogram.tsx

```diff
@@ -35,7 +30,7 @@ export default function MonthlyTotalHistogram({
   if (accounts && monthlyTotals) {
     guids.forEach(guid => {
       const data = dates.map((_, i) => (
-        monthlyTotals[i][guid]?.toNumber() || 0
+        monthlyTotals[i]?.[guid]?.toNumber() || 0
```

Comment:
this is needed now because eventual consistency between interval and monthlyTotals